src/main/java/org/mskcc/cbio/oncokb/web/rest/vm/usageAnalysis/ResourceModel.java

```diff
@@ -7,7 +7,7 @@
     
     public ResourceModel(String s){
         this.method =  s.split(" ")[0];
-        this.endpoint = s.split(" ")[1];
+        this.endpoint = s.split(" ")[1].replaceAll("\\%20", "");
```

Comment:
Replace the space escape to nothing?

Reply:
The target is removing unexpected space occurred in the endpoints, or sometimes endpoints will be shown like `/api/v1/annotate/%20%20%20%20%20%20%20%20%20%20%20%20%20mutations`